### DISCUSSION TITLE
Alarm bei verpassten Werten uberschreibt nicht den Lautlos Modus.

### DISCUSSION THREAD

**Discussion by elgrupo123** [NightscoutFoundation/xDrip#3789] [Bug reports]
Uns ist aufgefallen, daß es seit neuestem keine Alarme bei verpassten Werten auf dem Smartphone gibt, wenn es sich im lautlos Modus befindet. Der Alarm kommt nur, wenn das Handy nicht auf Stumm geschaltet ist. Das war früher definitiv nicht der Fall.

**Comment by Navid200** [comment]
Check the following and make sure it is enabled:  

`Settings` > `Alarms and Alerts` > `Other Alerts` > `Override Silent mode on these alerts`

**Comment by elgrupo123** [comment]
Ist aktiv.
![Screenshot_20241129-140058](https://github.com/user-attachments/assets/40a07368-d094-4020-ab4d-3cab05485d4a)

**Reply by Navid200** [reply]
OK

The missed reading alert, persistent high alert and forecast low alert are some of the alerts that do not use volume profile.
This means that if you lower the phone volume, the alert volume will be lowered.  If you reduce the volume to 0, the alert will make no sound and overriding silent mode will make no difference!

This is something that I intend to fix.  Although, I don't know when I will be able to as it is not easy:
https://github.com/NightscoutFoundation/xDrip/discussions/3697  

I'm not sure if this is what you have experienced.  But, it is a possibility.
It has always been like this.  This is not new.
